<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QMainWindow" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="stu_logout">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>注销登陆</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="stu_inq">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>查看课程信息</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="stu_choosecourse">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>选课</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="stu_person">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>查看个人课表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="stu_dele">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>退课</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="stu_signforass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>举手报名助教</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="stu_chooseass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>选择个人助教</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="7">
       <widget class="QStackedWidget" name="stackedWidget_3">
        <property name="currentIndex">
         <number>4</number>
        </property>
        <widget class="QWidget" name="page_s_1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QTableWidget" name="tableWidget_s_1"/>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>295</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="search_2"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_s_2">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>60</y>
            <width>330</width>
            <height>191</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>请输入原来的密码：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLineEdit" name="pass_past"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>输入要修改成的密码：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QLineEdit" name="pass_now"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>请再输入一遍：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QLineEdit" name="pass_nowa"/>
           </item>
           <item row="3" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>88</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QPushButton" name="modify_pas">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_s_3">
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>100</y>
            <width>401</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>请输入课程ID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="id_s_detele"/>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="detele_s">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>退课</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_s_4">
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>100</y>
            <width>401</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>请输入课程ID：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="signforass">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>报名此门课的助教！</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ass_s"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_s_5">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="6" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="2">
             <widget class="QPushButton" name="choose_ass">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>选择</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>请输入您想选择的助教：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="name_ass"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="Id_ass"/>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="search_ass">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>请输入课程ID：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QTextBrowser" name="display_ass"/>
            </item>
            <item row="0" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="idenline">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="timeline">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>个人中心</string>
    </property>
    <addaction name="actionb"/>
    <addaction name="actionc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actiona"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiona">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>退出系统</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
